index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+  <main>
+    <header>
```

Comment:
- [ ] KIndly reduce the spacing specified for your header section to match the figma design 👍🏽 

your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227382131-098d2b99-8a79-44d5-9396-d5b17c5d7f57.png)

figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/227382262-110a07c2-0554-480b-87ca-a65249c53bd0.png)